src/pages/index.js

```diff
@@ -1,88 +1,120 @@
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a41b184f-a3bd-4344-8359-a2d01db23146",
+    "Content-Type": "application/json",
   },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-];
+});
+
+api
+  .getAppInfo()
+  .then(([cards, user]) => {
+    cards.forEach((card) => {
+      const cardElement = getCardElement(card);
+      cardsList.prepend(cardElement);
+    });
+    profileName.textContent = user.name;
+    profileDescription.textContent = user.about;
+    profileAvatar.src = user.avatar; // Ensure the avatar URL is updated
+  })
+  .catch(console.error);
+
+/* -------------------------------------------------------------------------- */
+/*                            Modals for listeners                            */
+/* -------------------------------------------------------------------------- */
+const modals = document.querySelectorAll(".modal");
 
+/* -------------------------------------------------------------------------- */
+/*                              Profile elements                              */
+/* -------------------------------------------------------------------------- */
 const editModalButton = document.querySelector(".profile__edit-button");
 const cardModalButton = document.querySelector(".profile__add-button");
+const avatarModalButton = document.querySelector(".profile__avatar-button");
 const profileName = document.querySelector(".profile__name");
 const profileDescription = document.querySelector(".profile__description");
-
-const modals = document.querySelectorAll(".modal");
+const profileAvatar = document.querySelector(".profile__avatar");
 
 /* -------------------------------------------------------------------------- */
-/*                      Comment Divider = Shift + Alt + X                     */
+/*                          Avatar form elements                          */
 /* -------------------------------------------------------------------------- */
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarModalCloseButton = avatarModal.querySelector(
+  ".modal__close-button"
+);
+const avatarLinkInput = avatarModal.querySelector("#profile-avatar-input");
 
+/* -------------------------------------------------------------------------- */
+/*                      Edit form elements                                    */
+/* -------------------------------------------------------------------------- */
 const editModal = document.querySelector("#edit-modal");
 const editFormElement = editModal.querySelector(".modal__form");
 const nameInput = editModal.querySelector("#profile-name-input");
 const descriptionInput = editModal.querySelector("#profile-description-input");
+const submitButton = editFormElement.querySelector(".modal__submit-button");
```

Comment:
`submitButton`

There are 3 modals with different buttons and forms in the project. You need to **name** variables and functions the way all could understand what they are: `profileForm`, `profileCloseButton` and so on


`profileSubmitButton`

Reply:
* And this one https://github.com/ChrisLaFumee/se_project_spots/pull/1#discussion_r1985723580